packages/headless/.eslintrc.js

```diff
@@ -1,24 +1,25 @@
 module.exports = {
-  parser: '@typescript-eslint/parser',
-  parserOptions: {
-    project: 'tsconfig.json',
-    sourceType: 'module',
-  },
-  plugins: ['@typescript-eslint/eslint-plugin'],
-  extends: [
-    'plugin:@typescript-eslint/recommended',
-    'plugin:prettier/recommended',
-  ],
-  root: true,
-  env: {
-    node: true,
-    jest: true,
-  },
-  ignorePatterns: ['.eslintrc.js'],
-  rules: {
-    '@typescript-eslint/interface-name-prefix': 'off',
-    '@typescript-eslint/explicit-function-return-type': 'off',
-    '@typescript-eslint/explicit-module-boundary-types': 'off',
-    '@typescript-eslint/no-explicit-any': 'off',
-  },
+	parser: '@typescript-eslint/parser',
+	parserOptions: { 
+		project: 'tsconfig.json',
+    tsconfigRootDir: __dirname,
+		sourceType: 'module',
+	},
+	plugins: ['@typescript-eslint/eslint-plugin'],
+	extends: [
+		'plugin:@typescript-eslint/recommended', 
+	],
+	root: true,
+	env: {
+		node: true,
+		jest: true,
+	},
+	ignorePatterns: ['.eslintrc.js'],
+	rules: {
+		'@typescript-eslint/interface-name-prefix': 'off',
```

Comment:
config changes is not recommended unless necessary. for now you can remove this changes.

Reply:
this is not resolved yet. please don't change this file